README.md

```diff
@@ -66,6 +66,11 @@ and [Dynamic Color](https://m3.material.io/styles/color/dynamic-color/overview)
 
 # Download
 
+| Flavor      | Feature                          |
+|-------------|----------------------------------|
+| Default     | Android 8.0+, full support       |
```

Comment:
There's no animated webp support on api 26-28
They(Tenboro) haven't introduced that though.

Reply:
Will add that note when it's added.